custom/_legacy/pact/reports/chw_schedule.py

```diff
@@ -211,15 +213,15 @@ def chw_calendar_submit_report(request, username, interval=7):
 
     #secret date ranges
     if 'enddate' in request.GET:
-        end_date_str = request.GET.get('enddate', datetime.utcnow().strftime('%Y-%m-%d'))
-        end_date = datetime.strptime(end_date_str, '%Y-%m-%d')
+        end_date_str = request.GET.get('enddate', json_format_date(datetime.utcnow()))
+        end_date = iso_string_to_date(end_date_str)
     else:
-        end_date = datetime.utcnow()
+        end_date = datetime.utcnow().date()
 
     if 'startdate' in request.GET:
         #if there's a startdate, trump interval
-        start_date_str = request.GET.get('startdate', datetime.utcnow().strftime('%Y-%m-%d'))
-        start_date = datetime.strptime(start_date_str, '%Y-%m-%d')
+        start_date_str = request.GET.get('startdate', json_format_date(datetime.utcnow()))
+        start_date = iso_string_to_date(start_date_str)
         total_interval = (end_date - start_date).days
```

Comment:
`start_date` and `end_date` are only used here, so making both of them `date`s instead of `datetimes` works just fine, and is more sensical